ASUWebPlatforms/webspark-ci
# Ws2 2369 rfi updates

### Description

RFI component could allow multiple spam-clicks of the RFI form submit button to allow duplicate submissions. This updates the component to get the UDS updates to resolve that into place: https://github.com/ASU/asu-unity-stack/pull/1374

Additionally, we're including the update to the JSM link to the ERFIF form here. It was an 11th hour request that came in so the old form could be decommissioned.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-2369)
- [PR test site](https://pr-607-webspark-ci.ws.asu.edu/)

### Steps to Test

Submit the RFI form and confirm that after submit and before the success screen displays, that the submit button is disabled - preventing duplicate submissions. You can verify no duplicates make it through by visiting https://test-webspark-ci.ws.asu.edu/admin/reports/dblog?type%5B%5D=RFI+success  after doing an RFI submission and trying to trigger multiple submits. If there’s only one log entry for the time of that submit, the disabling worked.

Also, verify the “RFI Setup Request form” link on `/admin/config/asu_degree_rfi/settings` points to https://asudev.jira.com/servicedesk/customer/portal/2/create/286
